.circleci/devcluster/oauth.devcluster.yaml

```diff
@@ -29,6 +29,8 @@ stages:
           initial_user_password: $INITIAL_USER_PASSWORD
           authz:
             rbac_ui_enabled: true
+        telemetry:
```

Comment:
did they have the api key needed to report? https://github.com/determined-ai/determined/blob/3baece2ff30bc4cf3ec1f54bfce63902791882b0/master/pkg/config/telemetry.go#L6

Reply:
I have no idea. What's the impact either way?

Reply:
if it defaults to false, perhaps change the name of the PR to better reflect that
don't see an issue to merging anyway though

Reply:
It defaults to true. https://docs.determined.ai/latest/reference/deploy/master-config-reference.html#id29 

Infra noticed pretty quick growth in segment data coming from ci: https://hpe-aiatscale.slack.com/archives/C03JGLKTBEJ/p1720548565903099 

Good point on the API key; I don't know how it was successfully reported.